src/appkit.cs

```diff
@@ -760,6 +760,30 @@ interface NSApplication : NSAccessibilityElementProtocol, NSAccessibility {
 		void EnumerateWindows (NSWindowListOptions options, NSApplicationEnumerateWindowsHandler block);
 	}
 
+	[Static]
+	interface NSAboutPanelOption
```

Comment:
So the use of these is something like this:

			NSDictionary options = NSDictionary.FromObjectsAndKeys(new NSObject [] {(NSString)"Name", (NSString)"Version", new NSAttributedString("Credits")},
			                                                       new NSObject [] {NSAboutPanelOption.ApplicationName, NSAboutPanelOption.ApplicationVersion, NSAboutPanelOption.Credits}); 
			NSApplication.SharedApplication.OrderFrontStandardAboutPanelWithOptions2 (options);
(Needs https://bugzilla.xamarin.com/show_bug.cgi?id=57718 to build)

So if we were going to improve it, I'd assume we do so by adding a OrderFrontStandardAboutPanelWithOptions that takes some strongly typed item and not a dictionary of smart enums.